README.md

```diff
@@ -46,13 +48,11 @@ Typically sdc-vm-agent development is done by:
 
 ## Testing
 
-At the moment, sdc-vm-agent testing is done by running the VMAPI test suite.
-SSH into a running SDC and run the following commands:
+To run the vm-agent tests, go to the GZ and:
```

Comment:
@misterdjules it should work in the GZ of any SDC node I believe... Does adding that this must be SDC help? I'd have assumed SDC since sdc-vm-agent only exists in SDC.